src/common/types/clusters.ts

```diff
@@ -24,9 +24,18 @@ export type HostSubnet = {
   humanized: string;
 };
 export type HostSubnets = HostSubnet[];
-export type NetworkConfigurationValues = V2ClusterUpdateParams & {
-  hostSubnet?: string;
-  useRedHatDnsService?: boolean;
```

Comment:
this seems to be never used on networking as the prop is tied to cluster details. Maybe it is some leftover from the past ?